README.md

```diff
@@ -29,7 +29,7 @@ Beta:
 * https://beta.w3.org
 * https://beta.w3.org/_build_summary.json
 
-### Staging
+### Staging ***Not currently in use***
 * https://www-staging.w3.org
 * https://www-staging.w3.org/_build_summary.json
```

Comment:
We should probably remove the whole block rather than saying staging is not currently in use

Reply:
@JamieC24 can you also remove "(TBC)" from the heading